internal/types/webhook.go

```diff
@@ -18,11 +18,13 @@ type WebhookEvent struct {
 
 // Common webhook event names
 const (
-	WebhookEventInvoiceCreateDraft         = "invoice.create.drafted"
-	WebhookEventInvoiceUpdateFinalized     = "invoice.update.finalized"
-	WebhookEventInvoiceUpdatePayment       = "invoice.updated.payment"
-	WebhookEventInvoiceUpdateVoided        = "invoice.update.voided"
-	WebhookEventInvoiceAmountsRecalculated = "invoice.amounts.recalculated"
+	WebhookEventInvoiceCreateDraft          = "invoice.create.drafted"
+	WebhookEventInvoiceUpdateFinalized      = "invoice.update.finalized"
+	WebhookEventInvoiceUpdatePayment        = "invoice.updated.payment"
+	WebhookEventInvoiceUpdateVoided         = "invoice.update.voided"
+	WebhookEventInvoiceAmountsRecalculated  = "invoice.amounts.recalculated"
```

Comment:
This maybe we can skip as this will get triggered only when credit note is created so can actually have credit_note.created event rather than this